UI/Components/GameCode.razor

```diff
@@ -1,53 +1,57 @@
-@inject IJSRuntime JSRuntime
+@inject IJSRuntime JSRuntime;
 
-<h1 class="font-['Pacifico'] text-5xl focus:outline-none text-center mb-12">
-  <span class="text-[#d877aa]">Share</span> <span class="text-[#f59771]">Code</span>
-</h1>
+<div class="flex flex-col justify-between h-full">
+  <h1 class="font-['Pacifico'] text-5xl focus:outline-none text-center">
+    <span class="text-[#d877aa]">Share</span> <span class="text-[#f59771]">Code</span>
+  </h1>
 
-<div class="flex flex-col mb-4">
-  <label class="font-['Quicksand'] text-slate-200 mb-1.5 font-semibold" for="game-code">Unique Game Code</label>
+  <div class="flex flex-col">
```

Comment:
Can you undo all the spacing, please? Check your space settings in Rider for this file type.